package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap --hoist",
+    "install": "npm run update:core && lerna bootstrap --hoist",
```

Comment:
The `add-sibling` approach already has a simlilar problem (see the hack [here](https://github.com/jupyterlab/jupyterlab-google-drive/blob/master/.travis.yml#L14)).